helmnet/hybridnet.py

```diff
@@ -73,21 +82,38 @@ def init_f(self):
         nn_name = self.hparams.architecture
         if nn_name == "custom_unet":
             self.f = HybridNet(
-                activation_function=self.hparams.activation_function,
-                depth=self.hparams.depth,
-                domain_size=self.hparams.domain_size,
-                features=self.hparams.features,
-                inchannels=6,
-                state_channels=self.hparams.state_channels,
-                state_depth=self.hparams.state_depth,
+                activation_function = self.hparams.activation_function,
+                depth               = self.hparams.depth,
+                domain_size         = self.hparams.domain_size,
+                features            = self.hparams.features,
+                inchannels          = 6,
+                state_channels      = self.hparams.state_channels,
+                state_depth         = self.hparams.state_depth,
             )
+        # elif nn_name == "genetic_unet":
```

Comment:
shall we just take those various pieces of commented code out, until we add the code for the genetic algorithms?

Reply:
Yes, will remove it in next commit. On a side note, im going to create another branch with the full genetic search part included. That part will need more documentation before merging into master.

Reply:
Sounds great! To be honest, the whole code needs more documentation, I sit down one day and do so!